backend/controllers/container.rb

```diff
@@ -92,4 +92,16 @@ class ArchivesSpaceService < Sinatra::Base
     json_response(result)
   end
 
+
+  Endpoint.post('/repositories/:repo_id/top_containers/bulk/barcodes')
+  .description("Bulk update barcodes")
+  .params(["barcode_data", String, "JSON string containing barcode data {uri=>barcode}", :body => true],
```

Comment:
Without creating a JSONModel, I post the barcode data as a JSON string  { uri => barcode_value, uri_2 => barocde_value_2 }.  I know it's a little ugly.. so any ideas would be great!